src/python/WMCore/MicroService/MSOutput/MSOutput.py

```diff
@@ -310,6 +313,15 @@ def makeSubscriptions(self, workflow):
         # add a configurable weight value
         ruleAttrs["weight"] = self.msConfig['rucioDiskRuleWeight']
 
+        # Do not make subscription for relval workflow if not enabled
+        # and consider transfer as done
+        if workflow['IsRelVal'] and self.msConfig['enableRelValDisk']:
```

Comment:
`and not self.msConfig['enableRelValDisk']:`?

I would suggest not to use the word "subscription" in the message below as well, as subscriptions are different between the old PhEDEx and Rucio. Perhaps something like "Skipping disk output data placement..."